ocaml/xapi/xapi.ml

```diff
@@ -1058,6 +1058,10 @@ let server_init () =
           ; ("Initialising random number generator", [], random_setup)
           ; ("Initialise TLS verification", [], init_tls_verification)
           ; ("Running startup check", [], startup_check)
+          ; ( "Initialize cgroups via tgroup"
+            , []
```

Comment:
I will add a Xapi_globs.tgroups_enabled to guard the calls for thread classification.